metomi/isodatetime/main.py

```diff
@@ -49,12 +49,7 @@
 
     # 3.  Print ISO8601 duration as total amount of a unit
     # 3.1 Into the total number of hours (H), minutes (M) or seconds (S)
-    #     it represents. Note: negative durations should be escaped by telling
-    #     the command to stop processing more options with `--` or by adding a
-    #     backslash in front of the duration.
-    isodatetime --as-total=s PT1H
-    isodatetime --as-total=s -- -PT1H
-    isodatetime --as-total=s \\-PT1H
+    #     it represents.
```

Comment:
I don't think that there is a test for this functionality - but that's out of scope for this issue.

Reply:
Thanks Tim, Ive accepted those changes and put back the example that wasn't a negative duration - the negative durations have been added a bit further down. 
I added a test for checking the negative durations and these are all passing :)